ansible/roles/bootstrap/tasks/main.yml

```diff
@@ -1,8 +1,18 @@
 ---
 # tasks file for bootstrap
-- name: install kvm pre-requisite packages 
+- name: install virtualization tools
   yum:
```

Comment:
Should we be installing packages via dnf rather than yum? It looks like there is an ansible [dnf module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/dnf_module.html).

I think this is okay for now, but we should eventually convert this to dnf when we get the time (backlog item).

Reply:
We can move it to DNF in Q3. Don't want to make too many changes now as this needs to be implemented throughout.